### PR TITLE
i18n(es): Update of `configuration-reference.mdx`, `server-side-rendering.mdx` & `integrations-guide.mdx`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- What does this PR change? Give us a brief description.

Updated files as the following list:
  - `configuration-reference.mdx` -> 2c7e9f3d2a0d3548427495d7be30bb340ac13e98
  - `server-side-rendering.mdx` -> 60f35d65a6ef99bb13fafbbc312c332213ab5a70
  -  `integrations-guide.mdx` -> 60f35d65a6ef99bb13fafbbc312c332213ab5a70

### CODE REVIEW

File: src/content/docs/es/reference/configuration-reference.mdx [hunk 1]
```diff
@@ -391,6 +391,46 @@ Si el puerto dado ya está en uso, Astro probará automáticamente el siguiente
 ```
 
 
+### server.headers
+
+<p>
+
+**Tipo:** `OutgoingHttpHeaders`<br />
+**Por defecto:** `{}`<br />
+<Since v="1.7.0" />
```
Comment (vickvasquez) [inline src/content/docs/es/reference/configuration-reference.mdx:400]: ¿No debería traducirse esta parte?
```suggestion
<Desde v="1.7.0" />
```

File: src/content/docs/es/guides/server-side-rendering.mdx [hunk 2]
```diff
@@ -224,4 +224,5 @@ if (!product) {
 
 ### Server Endpoints
 
-Un server endpoint, también conocido como una **ruta API**, es un archivo `.js` o `.ts` dentro de la carpeta `src/pages/` que recibe una [Request](https://developer.mozilla.org/es/docs/Web/API/Request) y devuelve una [Response](https://developer.mozilla.org/en-US/docs/Web/API/Response). Una característica poderosa de SSR: Las rutas API permiten ejecutar código del lado del servidor de forma segura. Para aprender más consulta nuestra [guía de Endpoints](/es/core-concepts/endpoints/#endpoints-del-servidor-rutas-de-api).
+Un server endpoint, también conocido como una **ruta API**, es una función exportada desde un archivo `.js` o `.ts` dentro de la carpeta `src/pages/`. 
+La función recibe un [contexto del endpoint](/es/reference/api-reference/#contexto-del-endpoint) y devuelve una [Response](https://developer.mozilla.org/es/docs/Web/API/Response). Una característica poderosa de SSR, las rutas API son capaces de ejecutar código de forma segura en el servidor. Para aprender más, consulta nuestra [guía de Endpoints](/es/core-concepts/endpoints/#endpoints-del-servidor-rutas-de-api).
```
Comment (vickvasquez) [inline src/content/docs/es/guides/server-side-rendering.mdx:228]:
```suggestion
La función recibe un [contexto del endpoint](/es/reference/api-reference/#contexto-del-endpoint) y devuelve una [Response](https://developer.mozilla.org/es/docs/Web/API/Response). Una característica poderosa de SSR, las rutas API son capaces de ejecutar código de forma segura en el servidor. Para aprender más, consulta nuestra [guía de endpoints](/es/core-concepts/endpoints/#endpoints-del-servidor-rutas-de-api).
```

File: src/content/docs/es/reference/configuration-reference.mdx (hunk 1 above)
Comment (dreyfus92) [reply src/content/docs/es/reference/configuration-reference.mdx:400]: Ese no puede ser traducido por que el componente así se llama.

**Comment by dreyfus92** [comment]
Gracias por el review @vickvasquez 😁

**Comment by Yan-Thomas** [comment]
Thanks, @dreyfus92! Just a small detail: the Integrations Guide page `nav.ts` key (the title you see in the sidebar) also changed, can you please update it? 🙌

**Review by Yan-Thomas — APPROVED** [review]
LGTM, thanks for the contribution @dreyfus92 & @vickvasquez! 💜

**Pull request merged**